externs/shaka/cast.js

```diff
@@ -0,0 +1,56 @@
+
```

Comment:
When you define externs, you need to have a block comment containing `@externs` at the top.  Please copy the copyright header and `@externs` block comment from another extern file.

Without that, this file will not be seen as an "external" interface from the environment, and pieces of the implementation can still be renamed and inlined.  I see you are using `['foo']` style to avoid renaming, but inlining can't be defeated in this way.

When you fix the externs header, you should see those issues go away.